docs/setup.rst

```diff
@@ -342,6 +342,7 @@ and mod_wsgi, with a Paperless installation in ``/home/paperless/``:
         WSGIScriptAlias / /home/paperless/paperless/src/paperless/wsgi.py
         WSGIDaemonProcess example.com user=paperless group=paperless threads=5 python-path=/home/paperless/paperless/src:/home/paperless/.env/lib/python3.6/site-packages
         WSGIProcessGroup example.com
+				WSGIPassAuthorization On
```

Comment:
```suggestion
        WSGIPassAuthorization On
```

The remaining content is indented using spaces, whereas the added line uses tabs.